distribution/docs/src/main/resources/content/_using/using-catalog-search-ui.adoc

```diff
@@ -6,130 +6,149 @@
+Temporal:: An example of a time-range search.
 
 ==== Configuring a Workspace in ${catalog-ui}
 
-Configure each workspace with queries and share options.
+Configure each workspace with searches and share options.
 
 .Adding searches
-. Select the default name of the workspace (*New Workspace* or template name) to change to a unique name.
-. If *Blank* workspace was selected, select *Add Search* to add the first search. Each workspace can have up to ten searches.
-.. Change the name of the search.
+. From the default *Workspaces* view, select the workspace to add a search to.
+. Click *Search ${branding} Intrigue* in the upper left corner, enter search terms, and click *Search* to add a search. This step can be repeated to add additional searches. Each workspace can have up to ten searches.
 .. Select *Basic* Search to select simple search criteria, such as *text*, *time*, and *location*.
-.. Select *Advanced* Search to access a query-builder for more complex queries.
+.. Select *Advanced* Search to access a query builder for more complex queries.
+. Click the save (image:save-icon.png[save icon]) icon next to the workspace title in the upper left corner. 
 
-.Navigator Menu Options
+.Navigation Menu Options
 * *Workspaces*: View all available workspaces.
-* *Upload*: Add new resources to catalog.
-* *Open Workspaces*: [Workspace Name] (if applicable) Navigate to the [Workspace Name] workspace. Each open workspace is listed in this section of the menu.
+* *Upload*: Add new metadata and resources to the catalog.
+* *Sources*: Lists all sources and their statuses.
+* *Open Workspaces*: Lists open workspaces.
 
 .Workspace Menu Options
-* Select the *Options* button (image:options-icon.png[options icon]) next to the Workspace name.
+* To view a workspace's options from the *Workspaces* view, press the *Options* button (image:options-icon.png[options icon]) for the workspace.
 ** *Save*: Save changes to the workspace.
 ** *Run All Searches*: Start all saved searches within this workspace.
 ** *Cancel All Searches*: Cancel all running searches.
 ** *Open in New Tab*: Opens this workspace in a separate tab.
-** *View Sharing*: View and edit settings for sharing this workspace. (User must be signed in to share or view shared workspaces.)
-** *View Details*: View the current details for a cloud-based workspace (User must be signed in).
+** *View Sharing*: View and edit settings for sharing this workspace. Users must be signed in to share workspaces or view shared workspaces.
+** *View Details*: View the current details for a cloud-based workspace Users must be signed in to view workspace details.
 ** *Duplicate*: Create a copy of this workspace.
-** *Subscribe*: Receive email notifications for search results on this workspace (If email notifications have been enabled).
+** *Subscribe*: Receive email notifications for search results on this workspace. This requires email notifications to be enabled.
```

Comment:
❓ Do we have instructions to enable notifications? Might be a good idea to link them from here.

Reply:
I changed the description a bit. Subscribing and Unsubscribing is that same as enabling and disabling email notifications.